modin/experimental/pandas/test/test_io_exp.py

```diff
@@ -136,7 +137,13 @@ def test_read_csv_without_glob(self):
 
     def test_read_csv_glob_4373(self):
         columns, filename = ["col0"], "1x1.csv"
-        pd.DataFrame([[1]], columns=columns).to_csv(filename)
+        df = pd.DataFrame([[1]], columns=columns)
+        with (
+            warns_that_defaulting_to_pandas()
```

Comment:
why Dask warns here btw?

Reply:
`to_csv` is still implemented only for Ray. It uses communication between processes, I'm not sure if it's so easy to transfer to Dask, so I wanted to deal with it separately.